### PR TITLE
Update sharepoint-online-and-onedrive-migration-speed.md

### PR DESCRIPTION
**Pull request by JeremyEsposito**
Added a Question/Answer in regards to a SPMT 'design limitation' confirmed by the PG.
SPMT does not support attachment >100Kb in SP2010 OneNote today. To support it, the tool needs some design change. I resolved (escalation) as by design, and use the ADO item as PBI to do further investigation.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: migration/sharepoint-online-and-onedrive-migration-speed.md [hunk 1]
```diff
@@ -201,6 +201,10 @@ Answer: If you are experiencing a high volume of HTTP 503 responses blocking you
     - Which migration tool you are using. For example, SPMT, ShareGate, Mover, etc.
 </br></br>
 
+**Question: When migrating OneNote notebooks that contain attachments from SharePoint 2010 to SharePoint Online, all attachments greater than 100Kb are missing. ") ?**</br>
```
Comment (JohanFreelancer9) [inline migration/sharepoint-online-and-onedrive-migration-speed.md:204]:
```suggestion
**Question: When migrating OneNote notebooks that contain attachments from SharePoint 2010 to SharePoint Online, all attachments greater than 100 KB are missing.**</br>
```

File: migration/sharepoint-online-and-onedrive-migration-speed.md [hunk 2]
```diff
@@ -201,6 +201,10 @@ Answer: If you are experiencing a high volume of HTTP 503 responses blocking you
...
+Answer: In SharePoint 2010, OneNote notebooks with attachments greater than 100Kb are put into a folder with a special content type which the SharePoint Migration Tool cannot read.
+As a workaround, you can migrate your SharePoint 2010 data to SharePoint 2016, then use the SharePoint Migration Tool migrate the data from SharePoint 2016 to SharePoint Online.
```
Comment (JohanFreelancer9) [inline migration/sharepoint-online-and-onedrive-migration-speed.md:206]:
```suggestion
**Question: When migrating OneNote notebooks that contain attachments from SharePoint 2010 to SharePoint Online, all attachments greater than 100 KB are missing.**</br>
Answer: In SharePoint 2010, OneNote notebooks with attachments greater than 100 KB are put into a folder with a special content type that the SharePoint Migration Tool cannot read.
As a workaround, you can migrate your SharePoint 2010 data to SharePoint 2016, then use the SharePoint Migration Tool to migrate the data from SharePoint 2016 to SharePoint Online. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @JeremyEsposito, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**